dotnet/simple-app/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿/*
+Copyright 2022 CodeNotary, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Text;
+using ImmuDB;
+using ImmuDB.Exceptions;
+
+namespace simple_app;
+
+class Program
+{
+    public static async Task Main(string[] args)
+    {
+        await InitializeExample();
+        await AnotherInitializeExample();
+        await YetAnotherInitializeExample();
+        await MiscFunctionsUsageExample();
+        await ImmuClient.ReleaseSdkResources();
+    }
+
+    private static async Task InitializeExample()
+    {
+        var client = ImmuClient.NewBuilder().Build();
+        await client.Open("immudb", "immudb", "defaultdb");
+
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.Set(key, "");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = await client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.Value.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+
+        await client.Close();
+    }
+    
+    private static async Task AnotherInitializeExample()
+    {
+        var client = await ImmuClient.NewBuilder().Open();
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.Set(key, "");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = await client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+
+        await client.Close();
+    }
+
+    private static async Task YetAnotherInitializeExample()

```

Comment:
would be better to have some other name for the example. What it initializes ? entry, connection ?

Reply:
renamed as `OpenConnectionExample`. Also, added Readme.md where the functions from the sample project are described.